target/linux/ramips/image/mt7620.mk

```diff
@@ -5,6 +5,8 @@
 DEVICE_VARS += TPLINK_FLASHLAYOUT TPLINK_HWID TPLINK_HWREV TPLINK_HWREVADD TPLINK_HVERSION \
 	DLINK_ROM_ID DLINK_FAMILY_MEMBER DLINK_FIRMWARE_SIZE DLINK_IMAGE_OFFSET
 
+ATH_SOC := mt7620
```

Comment:
You can add a Device/Default in this file and override this one in default device profile template instead.

Reply:
Ah... I'm not sure if what I've said above actually works.

Reply:
I wonder whether mt7620 is enough here or whether one would distinguish between those mt7620a and mt7620n (which would require at least some device-specific SOC variables as in ath79 again)